backend/src/seed/factories/index.js

```diff
@@ -128,6 +128,22 @@ export default function Factory(options = {}) {
       this.lastResponse = await cleanDatabase({ driver: this.neo4jDriver })
       return this
     },
+    async emote({ to, data }) {
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-08-06T20:26:16Z" title="Tuesday, August 6th 2019, 10:26:16 pm +02:00">Aug 6, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -128,6 +128,23 @@ export default function Factory(options = {}) {
       this.lastResponse = await cleanDatabase({ driver: this.neo4jDriver })
       return this
     },
+    async emote({ from, to, data }) {
```


If you use `neode` you wouldn't have to make a HTTP request over the wire.